plugins/tech-insights/src/plugin.ts

```diff
@@ -54,6 +54,18 @@ export const ScorecardInfo = techInsightsPlugin.provide(
   }),
 );
 
+/**
+ * @public
+ */
+export const FactDetails = techInsightsPlugin.provide(
```

Comment:
make sure that you've run `yarn build:api-reports` and committed the results of that